migrations/20160822191515-processedAt-in-Donation-migration.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    return queryInterface.addColumn(
+      'Donations',
+      'processedAt',
```

Comment:
processedAt is mostly another check to let us know when it happened. Since we are often messing with the data manually in Postico, I haven't been trusting `updatedAt` very much.